### PR TITLE
add lts support schedule, fix outdated version info

### PR DESCRIPTION
**Pull request by jenweber**
The Builds site lists 2.18 as LTS.

I fixed this and added a table for lts support timelines. It's not the greatest that it's hard coded, but it's mvp.

Someone should check over my work by looking at past blog posts to see if the dates are accurate. It got a little confusing by the end.

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -18,6 +18,87 @@
   version becomes the new stable release of Ember.
 </p>
 
+<p>
+  LTS (Long Term Support) releases are done every four minor versions, plus the
+  first and last releases of major versions. An LTS release receives security
```
Comment (rwjblue) [inline app/templates/index.hbs]: not the first and last, only the last (specifically neither 2.0 nor 3.0 were ever considered for the LTS candidates).

File: app/fixtures/ember/lts.js [hunk 2]
```diff
@@ -4,11 +4,11 @@ export default {
   projectFilter: [/ember\./, /ember-template-compiler/],
   projectRepo: 'emberjs/ember.js',
   initialVersion: "2.18.0",
-  initialReleaseDate: "2018-01-01",
-  lastRelease: "2.18.2",
-  futureVersion: "2.18.3",
+  initialReleaseDate: "2018-08-27",
```
Comment (rwjblue) [inline app/fixtures/ember/lts.js]: This should be the first date of a 2.18 release, what is 2018-08-27?

**Comment by jenweber** [comment]
I have made the requested changes, plus added some language to indicate the one-week release window.

File: app/templates/index.hbs [hunk 3]
```diff
@@ -12,11 +12,103 @@
...
+      <td>
+        May 2019
+      </td>
+      <td>
+        Oct 2019
+      </td>
+    </tr>
+    <tr>
+      <td>
+        3.0
```
Comment (kategengler) [inline app/templates/index.hbs]: I am pretty sure that 3.0 is NOT an LTS release

**Comment by jenweber** [comment]
I think this is ready to merge now.

**Comment by rwjblue** [comment]
Thanks @jenweber!

**Pull request merged**